docs/kubernetes/user-guides/driver-install.md

```diff
@@ -20,4 +20,21 @@ $ ./deploy/setup-project.sh
 $ GCE_PD_SA_DIR=/my/safe/credentials/directory    # Directory to get the service account key
 $ GCE_PD_DRIVER_VERSION=stable                    # Driver version to deploy
 $ ./deploy/kubernetes/deploy-driver.sh
-```
\ No newline at end of file
+```
+
+## Disabling particular CSI driver services
+
+Traditionally, you run the CSI controllers with the GCE PD driver in the same Kubernetes cluster.
+Though, there may be cases where you will only want to run a subset of the available driver services (for example, one scenario is running the controllers outside of the cluster they are serving (while the GCE PD driver still runs inside the served cluster), but there might be others scenarios).
+The CSI driver consists out of these services:
+
+* The **controller** service starts the GRPC server that is used for communication between the CSI components. It is depending on the GCP service account credentials.
+* The **identity** service is responsible to provide identity services like capability information of the CSI plugin.
+* The **node** service implements the various operations for volumes like creation, deletion, etc. It is depending on the GCE metadata service.
```

Comment:
node implements operations that are run locally from the node. For example `NodePublishVolume`, `NodeStageVolume`. It does *not* do operations like `Create` or `ControllerPublish`